### PR TITLE
Add CloudEventsTarget docs

### PR DESCRIPTION
**Pull request by odacremolbap**
Add CloudEventsTarget user facing docs.

### CODE REVIEW

File: docs/guides/cloudeventstarget.md [hunk 1]
```diff
@@ -0,0 +1,57 @@
+# Exporting CloudEvents
+
+The Trigermesh `CloudEventsTarget` API object is used to export Cloudevents to external systems via HTTP.
+
+## Configuring a CloudEventsTarget Object
+
+The CloudEventsTarget accepts parameters to set the endpoint where CloudEvents will be sent and authentication. When succesfuly created it consumes and forwards to the configured an HTTP endpoint.
```
Comment (jmcx) [inline docs/guides/cloudeventstarget.md:7]: "successfuly" => successfully

File: docs/guides/cloudeventstarget.md [hunk 2]
```diff
@@ -0,0 +1,57 @@
+# Exporting CloudEvents
+
+The Trigermesh `CloudEventsTarget` API object is used to export Cloudevents to external systems via HTTP.
```
Comment (jmcx) [inline docs/guides/cloudeventstarget.md:3]: "Trigermesh" => TriggerMesh 🤪 🤪 🤪 

"Cloudevents" => "CloudEvents"

File: docs/guides/cloudeventstarget.md (hunk 1 above)
Comment (jmcx) [reply docs/guides/cloudeventstarget.md:7]: Maybe can remove the "When succesfuly created it consumes and forwards to the configured an HTTP endpoint." sentence which is a bit awkward, maybe better place in the "usage" section?

File: docs/guides/cloudeventstarget.md [hunk 3]
```diff
@@ -0,0 +1,57 @@
...
+      endpoint: https://external.systen/mypath
+    ```
+
+### Configuring Credentials (Optional)
+
+If the external system requires [Basic Authentication](https://datatracker.ietf.org/doc/html/rfc2617) this component can be configured to use credentials by means of Kubernetes secrets.
+
+The credentials are defined under `spec.credentials.basicAuths`:
+
+!!! example "Credentials for 2 users"
```
Comment (jmcx) [inline docs/guides/cloudeventstarget.md:25]: I don't understand why this says 'for 2 users'? it seems to be one user/pass only no?

File: docs/guides/cloudeventstarget.md [hunk 4]
```diff
@@ -0,0 +1,57 @@
...
+          username: user
+          password:
+            valueFromSecret:
+              name: ce-target-password
+              key: password
+    ```
+
+## Using the CloudEventsTarget
+
+Given the configuration options depicted in the preceding sections we can create this example CloudEventsTarget by creating this object at a TriggerMesh cluster:
```
Comment (jmcx) [inline docs/guides/cloudeventstarget.md:39]: could be simplified to

"Lets now create a complete CloudEventsTarget and deploy it to our cluster."

File: docs/guides/cloudeventstarget.md [hunk 5]
```diff
@@ -0,0 +1,57 @@
...
+
+!!! example "Example CloudEventsTarget"
```
Comment (jmcx) [inline docs/guides/cloudeventstarget.md:41]: could we be a bit more pedagogical like in the other guides and recommend putting it in a "cloudeventstarget.yaml" file and show the kubectl apply command to deploy it?

**Review by jmcx — CHANGES_REQUESTED** [review]
Not my best review, just some tweaks!

File: docs/guides/cloudeventstarget.md (hunk 3 above)
Comment (odacremolbap) [reply docs/guides/cloudeventstarget.md:25]: copy pasta :-1:

**Review by jmcx — APPROVED** [review]

**Pull request merged**